schemas/com.google.ga4.enhanced-measurement/video_complete/jsonschema/1-0-0

```diff
@@ -0,0 +1,45 @@
+{
+        "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+        "self": {
+                "vendor": "com.google.ga4.enhanced-measurement",
+                "name": "video_complete",
+                "version": "1-0-0",
+                "format": "jsonschema"
+        },
+        "description": "A video_complete GA4 enhanced-measurement event.",
+        "type": "object",
+        "properties": {
+                "video_current_time": {
+                        "description": "The current time of the video.",
+                        "type": ["number", "null"]
+                },
+                "video_duration": {
```

Comment:
The video-events' properties seem safe to change too as you point out. (also thank you @greg-el for helping me figure this out). What changed is:

- `video_current_time` and `video_duration` changed to `integer`
- `video_percent` left as number with minimum and maximum (even though it seems it could be an integer, there is some doubt)

further resources:
- [blogpost by Simo Ahava](https://www.simoahava.com/analytics/the-youtube-video-trigger-in-google-tag-manager/)
- [youtube JS API mentioned in docs](https://developers.google.com/youtube/player_parameters#enablejsapi)